lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -37,7 +37,15 @@ defmodule ExUnit.CaptureLog do
   Returns the binary which is the captured output.
 
   This function mutes the `:console` backend
-  and captures any log messages sent to Logger.
+  and captures any log messages sent to Logger from the calling
+  processes. It is possible to ensure explicit log messages from other
+  processes are captured by waiting for their exit or monitor
+  signal. This does not guarantee to capture VM generated log messages
+  when a process is spawned using a low level `Kernel` spawn function (e.g.
```

Comment:
Grammatically, "process is spawned...raises" is awkward.  "process that is spawned...raises" reads better.